API_register/serializers.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import serializers
+
+from API_register.models import Announcements
+
+
+class AnnouncementGetSerializer(serializers.ModelSerializer):
```

Comment:
Ideally serializer should be the same for each type of request, unles there are no specific acceptance criteria